routes/routes.go

```diff
@@ -0,0 +1,14 @@
+package routes
+
+import (
+	"github.com/JoseLuisTorrentera/academy-go-q42021/controllers"
+	"github.com/gorilla/mux"
```

Comment:
Use best practice, separate your package from third parties. The Convension should be:
Import(
Core packages

Your packages

Third party packages)